src/devnet/config/devnet/genesis-shelley.json

```diff
@@ -19,7 +19,7 @@
         "extraEntropy": {
             "tag": "NeutralNonce"
         },
-        "keyDeposit": 0,
+        "keyDeposit": 2000000,
```

Comment:
This is so devnet/integration tests can also test the proper unlocking of the stake key deposit when unregistering.